src/acl/mod.rs

```diff
@@ -280,13 +280,50 @@ impl AccessControl {
     /// Check if domain name is in proxy_list.
     /// If so, it should be resolved from remote (for Android's DNS relay)
     pub fn check_host_in_proxy_list(&self, host: &str) -> Option<bool> {
-        // Addresses in proxy_list will be proxied
-        if self.white_list.check_host_matched(host) {
-            return Some(true);
-        }
-        // Addresses in bypass_list will be bypassed
-        if self.black_list.check_host_matched(host) {
-            return Some(false);
+        info!("host checking: {}", host);
+        match self.mode {
+            Mode::BlackList => { //"[accept_all]" | "[proxy_all]"
```

Comment:
Code style:

1. this line of comment should be in a separated line
2. an empty space is required after `//`

```rust
// "[accept_all]" | "[proxy_all]"
Mode::BlackList => {
```

Reply:
an empty space after `//` is not added yet.